### PR TITLE
Provides example output and troubleshooting info for the pl-heudiconv…

### PR DESCRIPTION
**Pull request by bredamc**
… plugin

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -119,6 +119,49 @@ Run unit tests:
         -v /home/my_name/my_project/my_output/heudiconv_output:/out:z \
         quay.io/rh-impact/pl-heudiconv pl-heudiconv /in /out
 
+### Example output
+
+
+[subs=+quotes]
+....
```
Comment (grdryn) [inline README.adoc:126]: I think showing the output of running againstthe following directory from upstream might be more complete, WDYT? https://github.com/nipy/heudiconv/tree/master/heudiconv/tests/data/01-fmap_acq-3mm

**Review by computate — APPROVED** [review]
@grdryn @bredamc @rudolphpienaar  Thank you for making these updates, I tried it out and it's working for me! This is an excellent contribution to the ChRIS Project!

**Review by grdryn — APPROVED** [review]
Sorry for the delay @bredamc, I was afk for the past ~10 days. Looks good to me, so I'll merge. Thanks! :pray:

**Comment by bredamc** [comment]
Fab -- thanks, @grdryn!

**Pull request merged**